Makefile

```diff
@@ -1,94 +0,0 @@
-VIRTUALENV = virtualenv
```

Comment:
Fixed 49a3ef3. Generating coverage reports for multiple Go packages is a bit complicated so I'll leave that for later.